### ISSUE TITLE
Mac OS: Keyboard shortcuts using delete

### ISSUE THREAD

**Issue by tbdalgaard** [jcsteh/osara#661]
Hi!

Just noticed that OSARA keymap for Mac OS defaults to using the numpad delete key instead of the regular delete key. That means people on laptops will need to press fn-delete on their computers to get the same functionality.
Is it possible to either duplicate the actions so both delete and numpad delete works or changing this so the regular delete key is used.

In Reaper the key is refered to as backspace, or-though that is a slidely incorrect. The key works as backspace does, but for almost all Mac users there is not a backspace key on their keyboard - only a delete key.

I will be happy to post keymap files if that can help.

**Comment by GarthHum** [comment]
I don’t think we should be changing this one. Yes, this means that we need to press both FN and delete. 
Even though the key is called Delete, it works in the same way as backspace, and is in the same location typically.

**Comment by tbdalgaard** [comment]
@GarthHum wrote:

> Even though the key is called Delete, it works in the same way as backspace, and is in the same location typically. 

That's correct, but this is not a typical standard Mac OS way of approaching this. In all programs I have seen delete is refered to as the key above the return key. I know that most of us can translate this when working with Reaper, but working on a laptop where you have to press FN-delete to get the expected functionality feels very different and strange to me. Therefor my suggestion about doing a duplicate on the delete key, so we get the best from both worlds.  

The delete key we are using now is in fact called delete forward on Mac OS keyboards and is not found on any laptop these days.

If this isn't going to be changed, I think we have to mention this explicitly in any introduction material for Reaper and OSARA, so new users aren't expecting the delete key to work, but instead are forced to, either remap this themselves or using the FN-key quite freequently.

**Comment by GarthHum** [comment]
Definitely agree it should be Clare in any training. I do typically use a laptop keyboard, and  in so use the FN key to turn what Mac  calls delete key into The forward delete key. 
This doesn’t feel very different conceptually to me then our use of the command key instead of the Mac control key for all the cut-and-paste type actions.
    


Sent from my iPhone

On 31 Jan 2022, at 2:56 am, Thomas Byskov Dalgaard ***@***.***> wrote:

﻿
@GarthHum wrote:

Even though the key is called Delete, it works in the same way as backspace, and is in the same location typically.

That's correct, but this is not a typical standard Mac OS way of approaching this. In all programs I have seen delete is refered to as the key above the return key. I know that most of us can translate this when working with Reaper, but working on a laptop where you have to press FN-delete to get the expected functionality feels very different and strange to me. Therefor my suggestion about doing a duplicate on the delete key, so we get the best from both worlds.

The delete key we are using now is in fact called delete forward on Mac OS keyboards and is not found on any laptop these days.

If this isn't going to be changed, I think we have to mention this explicitly in any introduction material for Reaper and OSARA, so new users aren't expecting the delete key to work, but instead are forced to, either remap this themselves or using the FN-key quite freequently.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
